symplyphysics/laws/thermodynamics/volume_of_heated_liquid_from_initial_volume_and_heating_temperature.py

```diff
@@ -0,0 +1,42 @@
+from sympy import (Eq, solve)

```

Comment:
This law is not limited to liquids. Also solid bodies expand when heated (unless they melt).